Code/GraphMol/DistGeomHelpers/Wrap/testDistGeom.py

```diff
@@ -375,16 +375,24 @@ def test8MultiThreadMultiConf(self):
             ee = ff.CalcEnergy()
             energies.append(ee)
 
-        mol = Chem.AddHs(Chem.MolFromSmiles("CC(C)(C)c(cc12)n[n]2C(=O)/C=C(N1)/COC"))
-        cids = rdDistGeom.EmbedMultipleConfs(mol,200,maxAttempts=30,randomSeed=100,numThreads=4)
+        mol2 = Chem.AddHs(Chem.MolFromSmiles("CC(C)(C)c(cc12)n[n]2C(=O)/C=C(N1)/COC"))
+        cids2 = rdDistGeom.EmbedMultipleConfs(mol2,200,maxAttempts=30,randomSeed=100,numThreads=4)
+        self.assertTrue(lstEq(cids, cids2))
         nenergies = []
-        for cid in cids:
-            ff = ChemicalForceFields.UFFGetMoleculeForceField(mol, 10.0, cid)
+        for cid in cids2:
+            ff = ChemicalForceFields.UFFGetMoleculeForceField(mol2, 10.0, cid)
             ee = ff.CalcEnergy()
             nenergies.append(ee)
 
-        self.assertTrue(lstEq(energies, nenergies,tol=1e-6))
```

Comment:
Modified the PR to include system/compiler info in the build. While an `#ifdef` would suffice for C++, checking the compiler from Python wouldn't be possible. How do you like that? I think it could be useful also for other purposes.